django/template/defaulttags.py

```diff
@@ -1167,6 +1167,55 @@ def now(parser, token):
     return NowNode(format_string, asvar)
 
 
+@register.simple_tag(takes_context=True)
+def query_string(context, query_dict=None, **kwargs):
+    """
+    Add, remove, and change parameters of a `QueryDict` and return the result
+    as a querystring. If the `query_dict` argument is not provided,
+    `request.GET` will be used.
```

Comment:
Double backticks as in theory this is reStructuredText?

```suggestion
    Add, remove, and change parameters of a ``QueryDict`` and return the result
    as a query string. If the ``query_dict`` argument is not provided,
    ``request.GET`` will be used.
```